doc/reference/network_macvlan.md

```diff
@@ -9,6 +9,11 @@ You can then assign IP addresses based on the randomly generated MAC addresses.
 The `macvlan` network type allows to specify presets to use when connecting instances to a parent interface.
 In this case, the instance NICs can simply set the `network` option to the network they connect to without knowing any of the underlying configuration details.
 
+```{note}
+If you are using a `macvlan` network, host-to-instance communication is not possible.
```

Comment:
Can we say something that indicates its not just host-to-instance but instance-to-host that is prevented.

How about:

> If you are using a `macvlan` network, communication between LXD host and instance(s) is not possible.

Reply:
Also could you make this section an include and add to the NIC sections for macvlan and ipvlan please?

Reply:
Reuse is difficult here since the content isn't exactly the same, but I added the statements. :)